src/Features/Core/Diagnostics/Analyzers/RudeEditUserDiagnosticAnalyzer.cs

```diff
@@ -24,7 +24,7 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         public override Task AnalyzeSyntaxAsync(Document document, Action<Diagnostic> addDiagnostic, CancellationToken cancellationToken)
         {
             // No syntax diagnostics produced by the EnC engine.  
-            return Task.FromResult(false);
+            return SpecializedTasks.EmptyTask;
```

Comment:
original code probably returned Task.FromResult(,,,) because there is no way to get completed task except doing Task.Run(()=> {})